sympy/printing/latex.py

```diff
@@ -2501,3 +2501,130 @@ def print_latex(expr, **settings):
     """Prints LaTeX representation of the given expression. Takes the same
     settings as ``latex()``."""
     print(latex(expr, **settings))
+
+
+def multiline_latex(lhs, rhs, terms_per_line=1, environment="align", use_dots=False, **settings):
+    r"""
+    This function generates a LaTeX equation with a multiline right-hand side
+    in either an ``eqnarray`` or ``align*`` environment.
```

Comment:
Below you list 3 environments including IEEEeqnarray but here only the other two.